src/main/java/ca/on/oicr/gsi/dimsum/util/reporting/reports/CaseTatReport.java

```diff
@@ -107,7 +107,11 @@ public Test getTest() {
                     .map(date -> date.format(DateTimeFormatter.ISO_LOCAL_DATE)).orElse(null);
               }),
               Column.forInteger("Release Days", x -> x.getCase().getReleaseDaysSpent()),
-              Column.forString("Completion Date", x -> getCompletionDate(x.getCase())),
+              Column.forString("Completion Date", x -> {
+                LocalDate date = DataUtils.getCompletionDate(x.getCase(), true);
+                return date != null ? date.format(DateTimeFormatter.ISO_LOCAL_DATE)
+                    : x.getCase().isStopped() ? "STOPPED" : null;
+              }),
```

Comment:
This report has been updated upstream. There's now a separate column to indicate whether the case is stopped, and this should show the completion date even if stopped